modules/wri_toc/config/schema/wri_toc.schema.yml

```diff
@@ -0,0 +1,7 @@
+block.settings.wri_toc_horizontal_bar_menu:
+  type: block_settings
+  label: 'Horizontal bar menu block'
+  mapping:
+    example:
```

Comment:
I updated this so the values match the config -- we could also have deleted it, but this is easy.